coordinator/src/bin/coordinator.rs

```diff
@@ -121,13 +127,75 @@ async fn main() -> Result<()> {
     let mut conn = pool.get().unwrap();
     run_migration(&mut conn);
 
-    let app = router(
-        Node {
-            inner: node,
-            pool: pool.clone(),
-        },
-        pool,
-    );
+    let node = Node {
+        inner: node,
+        pool: pool.clone(),
+    };
+
+    tokio::spawn({
+        let node = node.clone();
+        async move {
+            loop {
+                let mut conn = node.pool.get().unwrap();
+                match db::positions::Position::get_all_open_positions(&mut conn) {
+                    Ok(positions) => {
+                        let positions = positions
+                            .into_iter()
+                            .filter(|p| {
+                                p.position_state == PositionState::Open
+                                    && OffsetDateTime::now_utc().ge(&p.expiry_timestamp)
+                            })
+                            .collect::<Vec<Position>>();
+
+                        for position in positions.iter() {
+                            tracing::debug!(%position.expiry_timestamp, "Attempting to closed expired position with {}", position.trader);
```

Comment:
Nit: (not sure if this compiles) 

```suggestion
tracing::debug!(trader_pk=position.trader, %position.expiry_timestamp, "Attempting to close expired position");
```